src/oc_erchef/rebar.lock

```diff
@@ -36,8 +36,8 @@
        {ref,"c2a6b925ed94ab8a28ff49375782834033919190"}},
   0},
  {<<"ej">>,
-  {git,"git://github.com/seth/ej.git",
-       {ref,"40821f7ab3d8746e8573e8542ad452d2b346ed7f"}},
+  {git,"git://github.com/lbakerchef/ej.git",
```

Comment:
lets point this to the repo in the chef org.

Reply:
This was already changed previously.  Not sure why it's showing incorrectly here.